tests/test_advanced.py

```diff
@@ -341,38 +364,26 @@ def test_positive_sync_plan_enable(ansible_module):
 
     :steps:
         1. Run foreman-maintain advanced procedure run
+        sync-plans-disable
+        2. Run foreman-maintain advanced procedure run
         sync-plans-enable
 
     :expectedresults: procedure sync-plans-enable should work.
 
     :CaseImportance: Critical
     """
-    contacted = ansible_module.command(Advanced.run_sync_plans_enable())
+    contacted = ansible_module.command(Advanced.run_sync_plans_disable())
     for result in contacted.values():
         logger.info(result['stdout'])
+        assert result["rc"] == 0
         assert "FAIL" not in result['stdout']
-
-
-def test_positive_sync_plan_disable(ansible_module):
-    """Run sync-plans-disable using advanced procedure run
-
-    :id: 8690d875-6784-41f3-92cc-06e2bc8b5dc0
-
-    :setup:
-        1. foreman-maintain should be installed.
-
-    :steps:
-        1. Run foreman-maintain advanced procedure run
-        sync-plans-disable
-
-    :expectedresults: procedure sync-plans-disable should work.
-
-    :CaseImportance: Critical
-    """
-    contacted = ansible_module.command(Advanced.run_sync_plans_disable())
+        assert "Total 1 sync plans are now disabled." in result["stdout"]
```

Comment:
@jameerpathan111 this wont be good idea to make the automation system dependant. Do not assert the count. Good way will be asserting id of sync-plan in data.yml.

Reply:
> @jameerpathan111 this wont be good idea to make the automation system dependant. Do not assert the count. Good way will be asserting id of sync-plan in data.yml.

makes sense